.github/workflows/ci.yml

```diff
@@ -49,6 +49,7 @@ jobs:
           done
         ) &
     - run: nix --experimental-features 'nix-command flakes' flake check -L
+    - run: nix --experimental-features 'nix-command flakes' flake show --all-systems --json
```

Comment:
```suggestion
    - run: nix --experimental-features 'nix-command flakes' flake show --all-systems --json
    - run: nix --experimental-features 'nix-command flakes' flake check --all-systems --no-build
```

Reply:
I believe this doesn't work because we need IFD:
```
$ nix --experimental-features 'nix-command flakes' flake check --all-systems --no-build
error:
       … while checking flake output 'hydraJobs'
...
       error: path '/nix/store/gnq4bdsqzca6csbc05r9lb6b680nmnwj-tmux-3.4.drv' is not valid
```

Reply:
I tried it myself and saw the same thing. I force-pushed away that commit, @roberth, in the naive hope that we don't have to wait the ~1h for CI again.

Here is your commit as a backup, though: b7f7d4940bdb3b0ba65c8382e3cab17153aabe2c